src/extension.ts

```diff
@@ -214,12 +214,20 @@ class DiskSpace extends Resource {
 
 
 class ResMon {
-    private _statusBarItem: StatusBarItem = window.createStatusBarItem(StatusBarAlignment.Left);
+    private _statusBarItem: StatusBarItem;
     private _config: WorkspaceConfiguration;
     private _delimiter: string;
     private _updating: boolean;
 
     constructor() {
+        if(this._config.get('alignLeft'))
```

Comment:
nit spacing and mandatory braces

```
if (this._config.get('alignLeft'))  {
...
}
```

Reply:
Maybe to make this easier for contributors there could be a style file for automatic formatting?